packages/lesswrong/server/dialogues/cron.ts

```diff
@@ -64,3 +66,23 @@ addCronJob({
     }
   }
 })
+
+addCronJob({
+  name: 'archiveStaleDialogues',
+  interval: 'every 24 hours',
+  async job() {
+    const context = createAdminContext();
+    const staleCutoff = new Date(Date.now() - (autoArchiveCutoffDays * 24 * 60 * 60 * 1000));
+    const staleDialogues = await context.repos.posts.getStaleDialogues(staleCutoff);
```

Comment:
How many users/dialogues is this going to catch without them getting a notification some number of days in advance, the first time it runs?

Reply:
tracking this consideration and will implement some kind of fix